src/Math-Tests-Matrix/PMMatrixTest.class.st

```diff
@@ -737,6 +737,29 @@ PMMatrixTest >> testSkalarMultiplication [
 	self assert: r class equals: PMSymmetricMatrix
 ]
 
+{ #category : #tests }
+PMMatrixTest >> testStrassenProductWithMatrix [
+		
+	| aPMMatrix expected strassenProduct |
+	
+   aPMMatrix := PMMatrix rows: #( (1 2 3 4) (1 0 1 2) (2 3 1 4) (4 3 2 2)).
+	expected  := PMMatrix rows: #( (30 12 27 24) (12 6 11 10) (27 11 30 27) (24 10 27 33)).
+	strassenProduct := aPMMatrix transpose strassenProductWithMatrix: aPMMatrix.
+	self assert: expected  equals: strassenProduct.
```

Comment:
This, I think, has to be the other way around. `self assert: strassenProduct equals: expected`, which reads better.

I really like the variable naming. Thank you

Reply:
@hemalvarambhia I have changed the order of `strassenProduct` and `expected`. Thank you.